src/human-readable/runtime/utils.ts

```diff
@@ -223,3 +235,15 @@ export function splitParameters(
 
   return []
 }
+
+export function validateSolidityType(type: string) {
```

Comment:
Let's rename this and add a type predicate (for fun):

```suggestion
export function isSolidityType(type: string): type is Exclude<AbiType, SolidityArray> {
```

Reply:
Agreed :P